lib/ui/addon/styles/_list.scss

```diff
@@ -58,6 +58,12 @@
   }
 }
 
+.list.list--css-striping {
```

Comment:
This just seemed like the easiest way to get striping working again.

Reply:
Is there not an existing class for striping?

Reply:
We do. It’s based on passing a boolean to a row. I think I was struggling with this since the dataset is nested.